hardware/infiniband.py

```diff
@@ -12,15 +12,17 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-'''Fetch information about Infiniband cards.'''
+# Fetch information about Infiniband cards.
```

Comment:
This change is incorrect, please keep the module docstring.

Reply:
I will, thanks